File: packages/sdk-py/src/grantex/manifests/gmail.py
from grantex.manifest import ToolManifest, Permission

manifest = ToolManifest(
    connector="gmail",
    description="Gmail API",
    tools={
        "send_email": Permission.WRITE,
        "read_inbox": Permission.READ,
        "search_emails": Permission.READ,
        "get_thread": Permission.READ,
    },
)

File: packages/sdk-ts/src/manifests/gmail.ts
import { ToolManifest, Permission } from '../manifest.js';

export const gmailManifest = new ToolManifest({
  connector: 'gmail',
  description: 'Gmail API',
  tools: {
    send_email: Permission.WRITE,
    read_inbox: Permission.READ,
    search_emails: Permission.READ,
    get_thread: Permission.READ,
  },
});
